Rebuilt Documentation in Gitbook

AlinaWithAFace opened the pull request:
You can see the results [here](https://project-north-star.gitbook.io/project-north-star/), but basically I rebuilt all of the documentation to work with gitbook since it's a bit more user-friendly compared to READMEs scattered throughout folders.

AlinaWithAFace committed:
GitBook: [master] 12 pages and 7 assets modified

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] 6 pages modified

AlinaWithAFace committed:
GitBook: [master] 4 pages modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] 9 pages modified

AlinaWithAFace committed:
GitBook: [master] 5 pages and 17 assets modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] 2 pages and one asset modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] 17 pages modified

AlinaWithAFace committed:
GitBook: [master] 18 pages modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] one page modified

jselstad commented on SUMMARY.md:
```diff
@@ -0,0 +1,35 @@
+  * [Initial Release Assembly Guide](mechanical/initial-release-assembly-guide.md)
+  * [Initial Release 3D Printing Reference](mechanical/initial-release-3d-printing-reference.md)
+  * [Changelog](mechanical/changelog.md)
+* [Software](software.md)
```
Ideally the top-level pages are preserved as `README.md`s (so Github's built-in readme system continues to catch it).

AlinaWithAFace replied:
Done!

AlinaWithAFace replied:
Arg, sorry, only sort of done. I've tried changing it several times, but gitbook seems to keep reverting it when I make other changes 😕

jselstad commented on software.md:
```diff
@@ -0,0 +1,41 @@
+
+We have included a pre-built version of the internal calibration tool. We can make no guarantees about the accuracy of the process in DIY environments; this pipeline is built from multiple stages, each with multiple points of failure. Included in the .zip file are a python script for calibrating the calibration cameras, a checkerboard .pdf to be used with that, and Windows-based Calibrator exe, a ...
+
+## Friendlier video overview
```
Probably a bad on our end; could you change this to "Video Overview"? :)

AlinaWithAFace replied:
Ok!

jselstad reviewed (changes requested):
Wow, this looks great!  Gitbook is a huge visual upgrade over the standard Github readmes.

A couple questions:
- Is this missing a `book.json`?
- How is ownership of the pages managed with respect to gitbook.io?  My current impression is that we'll need to initialize a separate Gitbook.io instance from your staging deployment.

Once the pieces below and the merge conflicts are fixed (and @fmaurer approves), I see no reason not to merge this.

AlinaWithAFace commented:
I've only been working with gitbook for the past month or so using their [webapp](https://app.gitbook.com) client, so admittedly I don't know if it's missing a `book.json` or not, or what it would look like.
They apparently use some sort of teams system, again via their webapp. I'd be happy to add leap motion staff to what I've already set up if you'd like to keep the domains (I'd need emails!), or you should be able to connect another instance via their github integration.

AlinaWithAFace committed:
Merge branch 'master-origin'

# Conflicts:
#	Mechanical/CHANGELOG.md
#	Mechanical/README.md
#	README.md

AlinaWithAFace committed:
Merge branch 'master' of https://github.com/AlinaWithAFace/ProjectNorthStar

AlinaWithAFace committed:
dealing with dual folder case step 1

AlinaWithAFace committed:
dealing with dual folder step 2

AlinaWithAFace committed:
GitBook: [master] 11 pages modified

AlinaWithAFace committed:
GitBook: [master] 4 pages modified

AlinaWithAFace committed:
GitBook: [master] 26 pages modified

AlinaWithAFace committed:
folder structure and readmes

AlinaWithAFace committed:
GitBook: [master] 13 pages modified

AlinaWithAFace committed:
Update README.md

AlinaWithAFace committed:
removed redundant assets

AlinaWithAFace committed:
migrated BOM from excel to markdown

AlinaWithAFace committed:
Merge branch 'master' of https://github.com/AlinaWithAFace/ProjectNorthStar

# Conflicts:
#	SUMMARY.md

AlinaWithAFace committed:
GitBook: [master] 12 pages modified

AlinaWithAFace committed:
readme case

AlinaWithAFace committed:
fun fact git is very case sensetive

AlinaWithAFace committed:
so case sensetive you would not believe

AlinaWithAFace committed:
GitBook: [master] 12 pages and 3 assets modified

AlinaWithAFace committed:
added images

AlinaWithAFace committed:
GitBook: [master] 11 pages and 4 assets modified

AlinaWithAFace committed:
GitBook: [master] 4 pages modified

AlinaWithAFace committed:
GitBook: [master] no pages and 9 assets modified

AlinaWithAFace committed:
fixed broken articles

AlinaWithAFace committed:
GitBook: [master] 13 pages modified

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
Update software.md

changed recommendations to unity 2018.4 LTS as there is a bug in 2019.3 occasionally with matrix out of bounds. 

Added line for making sure the display is showing properly in display settings, as that is necessary for the unity code to work properly.

AlinaWithAFace committed:
Merge pull request #4 from BryanChrisBrown/patch-1

Unity version recommendation, extra setup step

AlinaWithAFace committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] 3 pages modified

AlinaWithAFace committed:
GitBook: [master] one page modified

Commit:
added steamvr documentation for real this time

Commit:
fixed syntax on steamvr.md

Commit:
more syntax updates

AlinaWithAFace committed:
Merge pull request #7 from BryanChrisBrown/master

added steamvr documentation to the software category

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
merge from BryanChrisBrown-patch-3

AlinaWithAFace committed:
Merge branch 'master' into alina-master

AlinaWithAFace committed:
Merge pull request #9 from AlinaWithAFace/alina-master

merge from BryanChrisBrown-patch-3

AlinaWithAFace committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 25 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
GitBook: [master] 6 pages modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
Update calibration-guide.md

BryanChrisBrown committed:
GitBook: [master] 19 pages modified

BryanChrisBrown committed:
GitBook: [master] 6 pages modified

BryanChrisBrown committed:
GitBook: [master] 28 pages and 3 assets modified

AlinaWithAFace committed:
Merge pull request #10 from BryanChrisBrown/patch-2

Update calibration-guide.md

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 27 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 27 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 26 pages and one asset modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 12 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

AlinaWithAFace committed:
Update README.md

BryanChrisBrown committed:
GitBook: [master] 26 pages modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
GitBook: [master] 30 pages and 3 assets modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages and one asset modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 32 pages and 157 assets modified

MatthewCroughan committed:
Note that #230-004 is now removed

It is currently undocumented that part #230-004 is removed. Though searching for this part number in the Discord shows that @BryanChrisBrown has mentioned it has been removed due to undesired cable catching.

BryanChrisBrown committed:
GitBook: [master] one page modified

AlinaWithAFace committed:
Merge pull request #14 from MatthewCroughan/patch-1

Note that #230-004 is now removed

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 5 pages and 4 assets modified

BryanChrisBrown committed:
GitBook: [master] 7 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 34 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 10 pages modified

BryanChrisBrown committed:
GitBook: [master] 6 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 33 pages modified

BryanChrisBrown committed:
GitBook: [master] one page and 2 assets modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 7 pages and one asset modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 7 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages and 12 assets modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 35 pages modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 9 pages and one asset modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page and one asset modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 11 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 37 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 2 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] 36 pages modified

BryanChrisBrown committed:
GitBook: [master] 3 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [master] one page and 2 assets modified

BryanChrisBrown committed:
GitBook: [master] 38 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

HyperLethalVector committed:
GitBook: [master] one page and 2 assets modified

HyperLethalVector committed:
GitBook: [master] one page and 7 assets modified

HyperLethalVector committed:
GitBook: [master] one page modified

HyperLethalVector committed:
GitBook: [master] one page modified

HyperLethalVector committed:
GitBook: [master] 37 pages modified

BryanChrisBrown committed:
GitBook: [master] one page modified

BryanChrisBrown committed:
GitBook: [#197] Remove links to smart prototyping web page

BryanChrisBrown committed:
GitBook: [#208] change subtitle

BryanChrisBrown committed:
GitBook: [#209] consolidate FAQ

BryanChrisBrown committed:
GITBOOK-212: change request with no subject merged in GitBook

BryanChrisBrown committed:
GITBOOK-214: Add Sensor Guide, fix spelling inconsitencies

BryanChrisBrown committed:
GITBOOK-213: Add NS Next to homepage and add Realsense version note

BryanChrisBrown committed:
GITBOOK-210: Change S-P store link to Ahead.IO

BryanChrisBrown committed:
GITBOOK-215: Update terminology on Northstar Next Assembly Guide
